src/index.js

```diff
@@ -59,9 +65,19 @@ function IpfsAPI (host_or_multiaddr, port) {
 
   // -- Interface
 
+  /** @public */
   self.send = requestAPI
 
-  self.add = (files, opts, cb) => {
+  /**
+   * Add a file/many files to IPFS returning the hash and name. The
+   * name value will only be set if you are actually sending a file.
+   *
+   * @public
+   * @param {} files
+   * @param {object} opts
+   * @param {function} cb
+   */
```

Comment:
**Pros**
- With the code
- less chance of out of sync

**Cons**
- looks like Java